src/Response.php

```diff
@@ -122,7 +122,7 @@ public function withStatus($code, $reasonPhrase = '')
     {
         $new = clone $this;
         $new->statusCode = (int) $code;
-        if (!$reasonPhrase && isset(self::$phrases[$new->statusCode])) {
+        if ($reasonPhrase == '' && isset(self::$phrases[$new->statusCode])) {
```

Comment:
It fixes string `'0'`. Integer `0` is not even allowed as argument. So that argument is pointless.